src/common/runtime/src/repeated_task.rs

```diff
@@ -204,7 +204,7 @@ mod tests {
         tokio::time::sleep(Duration::from_millis(550)).await;
         task.stop().await.unwrap();
 
-        assert_eq!(n.load(Ordering::Relaxed), 5);
+        assert!(n.load(Ordering::Relaxed) >= 3);
```

Comment:
This test is unstable so I relax the assertion.

Reply:
Sleep for more time may work.

Reply:
If we sleep more time, then we might need to increase the expected value passed to `assert_eq` and encounter the same issue. We can still ensure the repeated task executes repeatedly by checking `n >= 3`.